configuration/config.go

```diff
@@ -0,0 +1,51 @@
+package configuration
+
+import (
+	"fmt"
+	"github.com/joho/godotenv"
+	"github.com/spf13/viper"
+	"log"
+	"strconv"
+)
+
+type Config struct {
+	SMTPHost           string
+	SMTPPort           string
+	SMTPSender         string
+	SMTPPassword       string
+	ParallelEmailCount int
+	EmailsList         string
+	MessageFilePath    string
+	EmailSubject       string
+}
+
+func LoadConfig() (*Config, error) {
```

Comment:
```suggestion
func Load() (*Config, error) {
```
It's more clean to use the package like `configuration.Load` then `configuration.LoadConfig`.
Main idiom of coding is to avoid duplicate namings.